autoload/battery/backend/termux.vim

```diff
@@ -0,0 +1,19 @@
+" Ref: https://wiki.termux.com/wiki/Termux-battery-status
+let s:Job = vital#battery#import('System.Job')
+
+function! s:termux_update() abort dict
+  let self.is_charging = json_decode(system('termux-battery-status')).status !=# 'NOT_CHARGING'
+  let self.value = json_decode(system('termux-battery-status')).percentage + 0
+endfunction
```

Comment:
`system()` should not be used. Use `System.Job` instead.

You can find the documentation here
https://github.com/lambdalisue/vital-Whisky/blob/master/doc/Vital/System/Job.txt